samples/nrf9160/serial_lte_modem/src/main.c

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <logging/log.h>
+LOG_MODULE_REGISTER(app, CONFIG_SLM_LOG_LEVEL);
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <uart.h>
+#include <string.h>
+#include <bsd.h>
+#include <lte_lc.h>
+
+#include <hal/nrf_gpio.h>
+#include <hal/nrf_power.h>
+#include <hal/nrf_regulators.h>
+
+#include "slm_at_host.h"
+
+#if defined(CONFIG_SLM_CONNECT_UART_0) && defined(CONFIG_SLM_GPIO_WAKEUP)
+#error CONFIG_SLM_GPIO_WAKEUP should not be defined when UART0 is used
+#endif
+
+/* global variable used across different files */
+struct at_param_list m_param_list;
+
+/**@brief Recoverable BSD library error. */
+void bsd_recoverable_error_handler(uint32_t err)
+{
+	LOG_ERR("bsdlib recoverable error: %u", err);
+}
+
+/**@brief Irrecoverable BSD library error. */
+void bsd_irrecoverable_error_handler(uint32_t err)
```

Comment:
We have now removed the `bsd_irrecoverable_error_handler()`, from bsdlib.
This should be removed.

Reply:
Rebased to BSDlibv0.5.0 and have this function removed from main.c.
Thanks!